lib/matplotlib/axes/_base.py

```diff
@@ -3836,7 +3836,7 @@ def set_yscale(self, value, **kwargs):
 
         Parameters
         ----------
-        value : {"linear", "log", "symlog", "logit", ...}
+        value : {"linear", "log", "symlog", "logit", ...} or `.ScaleBase`
```

Comment:
Need to update docs for `set_xscale` as well?

Reply:
oops!.

Reply:
Maybe also 3D `set_zscale`? It only supports linear though.

Reply:
Yeah, so I guess not, since we don't want arbitrary scales going there?